frontend/store/collaterals.ts

```diff
@@ -98,12 +99,24 @@ export const actions = {
                     priceDropRatio: error.toString(),
                 };
             });
+            const autoRouteData = await fetchAutoRouteInformation(network, collateral.symbol).catch(error => {
+                console.error(error);
+                return {
+                    autoRouteError: error.toString(),
+                    autoRouteQuote: undefined,
+                    autoRouteExchanges: undefined,
+                };
+            });
```

Comment:
Nit: You can move `try/catch` into the `fetchAutoRouteInformation` itself, so that the code defining the keys stays in one place (in case you need to change something, you don't need to edit two places)